README.md

```diff
@@ -2,6 +2,8 @@
 
 # Icinga DB - Docker image
 
+**NOTE: With the release of Icinga DB [v1.3.0](https://github.com/Icinga/icingadb/releases/tag/v1.3.0), everything container-related was moved in the main repository.**
```

Comment:
GitHub has a very nice feature for this:

```suggestion
> [!NOTE]
> With the release of Icinga DB [v1.3.0](https://github.com/Icinga/icingadb/releases/tag/v1.3.0), everything container-related was moved in the main repository.**
```

Maybe just link the "main repository" so that it's absolutely clear which one this is.